backend/app/utils.py

```
@@ -444,27 +459,78 @@ def resolve_audio_base64(data: str, mime_type: str) -> tuple[str, str | None]:
+        if isinstance(pdf_input.content, list)
+        else [pdf_input.content]
+    )
+    for c in contents:
+        if not c.mime_type:
+            c.mime_type = "application/pdf"
+    return contents
+
+
+def resolve_input(
+    query_input,
+) -> tuple[str | list[ImageContent] | list[PDFContent] | "MultiModalInput", str | None]:
```

Ruff (0.15.2)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)